chapters/synchronous.tex

```diff
@@ -534,14 +534,14 @@ \section{Clocked State Variables}\label{clocked-state-variables}
 \end{tabular}
 \end{center}
 
-A variable to which \lstinline!previous! has been applied is called a \firstuse[clocked!state variable]{clocked state variable}\index{state variable!clocked}.
+A variable (i.e. not parameter) to which \lstinline!previous! has been applied is called a \firstuse[clocked!state variable]{clocked state variable}\index{state variable!clocked}.
 
 \begin{operatordefinition}[previous]
 \begin{synopsis}\begin{lstlisting}
 previous($u$)
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-The input argument $u$ is a component expression (\cref{def:component-expression}) or a parameter expression.
+The input argument $u$ is a component expression (\cref{def:component-expression}), that variable is then a clocked state variable, or a parameter expression.
```

Comment:
I get the point, but I think it needs to be formulated more carefully to avoid ambiguity for parameter component expressions:
```suggestion
The input argument $u$ is a component expression (\cref{def:component-expression}) or a parameter expression.
In case of a component expression which is not a parameter, $u$ is a clocked state variable according to the definition above.
```

Reply:
Clear in current formulation.  Resolving.